src/main/java/com/cleanroommc/groovyscript/compat/mods/aetherlegacy/Accessory.java

```diff
@@ -0,0 +1,88 @@
+package com.cleanroommc.groovyscript.compat.mods.aetherlegacy;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.accessories.AccessoryType;
+import com.gildedgames.the_aether.api.accessories.AetherAccessory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class Accessory extends ForgeRegistryWrapper<AetherAccessory> {
+
+    private static final AccessoryType[] VALUES = AccessoryType.values();
+
+    public Accessory() {
+        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOf("Accessory"));
+    }
+
+    @MethodDescription(type = MethodDescription.Type.ADDITION)
+    public void add(ItemStack item, String type) {
+        AccessoryType accessoryType = AccessoryType.MISC;
+        for (AccessoryType value : VALUES) {
+            if (value.getDisplayName().equalsIgnoreCase(type)) {
+                accessoryType = value;
+            }
+        }
+        AetherAccessory accessory = new AetherAccessory(item, accessoryType);
+        add(accessory);
+    }
+
+    @MethodDescription(description = "groovyscript.wiki.removeByInput", example = @Example("item('aether_legacy:iron_pendant')"))
+    public void removeByInput(ItemStack input) {
+        this.getRegistry().getValuesCollection().forEach(accessory -> {
+            if (accessory.getAccessoryStack().isItemEqual(input)) {
+                ReloadableRegistryManager.removeRegistryEntry(this.getRegistry(), accessory.getRegistryName());
+            }
+        });
+    }
+
+    @RecipeBuilderDescription(example = @Example(".input(item('minecraft:shield')).accessoryType('shield')"))
+    public RecipeBuilder recipeBuilder() {
+        return new RecipeBuilder();
+    }
+
+    @Property(property = "input", valid = @Comp("1"))
+    public static class RecipeBuilder extends AbstractRecipeBuilder<AetherAccessory> {
+
+        @Property
+        private AccessoryType accessoryType = AccessoryType.MISC;
+
+        @RecipeBuilderMethodDescription
+        public RecipeBuilder accessoryType(String type) {
+            for (AccessoryType value : VALUES) {
+                if (value.getDisplayName().equalsIgnoreCase(type)) {
+                    accessoryType = value;
+                }
+            }
```

Comment:
See above regarding enum values. Additionaly please add a method wich accepts the enum value directly.